src/taco-seasoning.md

```diff
@@ -0,0 +1,25 @@
+# Taco Seasoning
+
+Works great in any recipes that use taco seasoning. Makes about one store-bought packet of taco seasoning
+
+## Ingredients
+
+- 1 tbsp and 1.5 tsp Chili Powder
+- 1/2 tsp onion powder
+- 1/2 tsp garlic powder
+- 1/2 tsp seasoned salt
+- 1/2 tsp paprika
+- 1/2 tsp garlic salt
+- 1/4 tsp cumin
+- 1/4 tsp cayenne pepper
+- 1/4 tsp sugar
+
+## Directions
+
+1. Mix ingredients in a bowl
```

Comment:
These directions are incomplete.

Reply:
How? Once you mix the ingredients the seasoning is made. It's tagged as basic because it isn't stand alone.

Reply:
Well if that's the only direction, could you elaborate about mixing ingredients into the bowl? Like are you using any specific utensil or?

Reply:
I changed the wording in the latest commit to hopefully make it more clear.